iOS/Flows/Conversation/Cells/MessageSequenceCollectionViewDataSource.swift

```diff
@@ -127,24 +152,70 @@ extension MessageSequenceCollectionViewDataSource {
             cell.configure(with: message, showAuthor: false)
         }
     }
+
+    static func createLoadMoreCellRegistration() -> LoadMoreCellRegistration {
+        return LoadMoreCellRegistration { cell, indexPath, item in }
+    }
 }
 
 // MARK: - TimelineCollectionViewLayoutDataSource
 
 extension MessageSequenceCollectionViewDataSource: TimeMachineCollectionViewLayoutDataSource {
 
-    func getTimeMachineItem(forItemAt indexPath: IndexPath) -> TimeMachineLayoutItem? {
-        guard let item = self.itemIdentifier(for: indexPath) else { return nil }
-        let messageController = ChatClient.shared.messageController(cid: item.channelID,
-                                                                    messageId: item.messageID)
+    func getTimeMachineItem(forItemAt indexPath: IndexPath) -> TimeMachineLayoutItemType {
```

Comment:
There are now three different types of time machine items so this ordering logic got a little more complicated.

Messages just return their date created.
The placeholder is always at the front so return greatestFiniteMagnitude.
The load more is at the back.